src/ds/ds6/ds6-safety.cpp

```diff
@@ -49,6 +51,13 @@ namespace librealsense
         std::unique_ptr<frame_timestamp_reader> ds_timestamp_reader_metadata(new ds_timestamp_reader_from_metadata(std::move(ds_timestamp_reader_backup)));
 
         auto enable_global_time_option = std::shared_ptr<global_time_option>(new global_time_option());
+
+        synthetic_sensor& safety_sensor = dynamic_cast<synthetic_sensor&>(get_sensor(_safety_device_idx));
```

Comment:
I placed it in `register_options()` function,

I think it's better to aligned to that

Reply:
done